alchy/model.py

```diff
@@ -126,12 +125,7 @@ class User(Model):
     @declared_attr
     def __table_args__(cls):  # pylint: disable=no-self-argument
         # pylint: disable=no-member
-        args, kargs = merge_declarative_args(cls.__bases__, '__table_args__')
```

Comment:
Correct. I changed the behavior to only merge ``__local_table_args__`` from all ``cls.__mro__``. The problem isn't so much with attempting to merge ``__table_args__``, but rather than in many cases this is useless, as an attempt to set ``__table_args__`` will result in overridding ``ModelBase.__table_args__`` so that no merge is even attempted. So I changed the paradigm to be that one never touches ``__table_args__`` but rather simply sets ``__local_table_args__`` wherever one wants (final class; intermediate class; mixin).